externals/CMakeLists.txt

```diff
@@ -26,6 +26,11 @@ target_include_directories(stb INTERFACE stb)
 add_library(imgui INTERFACE)
 target_include_directories(imgui INTERFACE imgui)
 
+# Imguizmo
+add_library(imguizmo STATIC imguizmo/ImGuizmo.cpp)
```

Comment:
Leaving this review comment here, but it concerns the library dependency in general. ImGuizmo has been added directly to the project, not as a git submodule.